### PR TITLE
Add G3Constants namespace and python bindings

### PR DESCRIPTION
**Pull request by arahlin**
This commit provides a set of common mathematical and physical constants, stored
in G3Units, for use in creating code with consistent units handling.

### CODE REVIEW

File: core/include/core/G3Constants.h [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#include <G3Units.h>
+
+namespace G3Constants {
+	using namespace G3Units;
+
+	/* Math */
+	const double pi = M_PI;
+	const double twopi = 2.0 * M_PI;
+	const double halfpi = M_PI / 2.0;
+
```
Comment (nwhitehorn) [inline core/include/core/G3Constants.h:14]: These are already defined constants (even the last, e.g. M_PI_2) in C. Is there a good reason to have private names for them?

File: core/include/core/G3Constants.h [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#ifndef _G3_CONSTANTS_H
+#define _G3_CONSTANTS_H
+
+#include <cmath>
+#include <G3Units.h>
+
+namespace G3Constants {
+	using namespace G3Units;
```
Comment (nwhitehorn) [inline core/include/core/G3Constants.h:8]: Does this end up making G3Constants::K work by accident?

File: core/include/core/G3Constants.h (hunk 1 above)
Comment (arahlin) [reply core/include/core/G3Constants.h:14]: No real reason other than completeness, although the list isn't really complete anyway.  I'm fine with removing them.

File: core/include/core/G3Constants.h (hunk 2 above)
Comment (arahlin) [reply core/include/core/G3Constants.h:8]: Hmm possibly, haven't checked.  Is there a more correct way to do this, other than having to explicitly spell out `G3Units::` everywhere?

File: core/include/core/G3Constants.h (hunk 1 above)
Comment (nwhitehorn) [reply core/include/core/G3Constants.h:14]: I think I would prefer to drop them if you don't mind.

File: core/include/core/G3Constants.h (hunk 2 above)
Comment (nwhitehorn) [reply core/include/core/G3Constants.h:8]: After a quick test, it does make G3Constants::K work. I am not aware of an alternative except spelling out G3Units::.

File: core/include/core/G3Constants.h (hunk 1 above)
Comment (arahlin) [reply core/include/core/G3Constants.h]: Done.

File: core/include/core/G3Constants.h (hunk 2 above)
Comment (arahlin) [reply core/include/core/G3Constants.h]: Okie doke

**Review by nwhitehorn — APPROVED** [review]

**Pull request merged**